duffy/models/opennebula_nodes.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+Implementation of OpenNebula node model.
+
+https://opennebula.io/
+"""
+from duffy.database import db, Duffyv1Model
+
+
+class OpennebulaHost(Duffyv1Model):
+    """
+    Open Nebula node model.
+
+    Attributes:
+      id (int): Unique identifier of the node
+      name (str): Open Nebula VM name
+      hostname (str): Node hostname
+      ip (str): Node IP address
+      state (str): Current state of the node
+      next_state (str): Future state of the node
+      comment (str): Comment for this node. For example output of the failed test.
+      template_id (str): Identifier of the template used for VM creation
+      flavor (str): Type of the VM based on the resources
+      session_id (str): Id of the session linked to this machine
+      session (`db.relationship`): Relationship to `Session` table
+    """
+    __tablename__ = 'opennebula_nodes'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(30))
+    hostname = db.Column(db.String(20))
+    ip = db.Column(db.String(15))
+    state = db.Column(db.String(20))
+    next_state = db.Column(db.String(20))
```

Comment:
@Zlopez let's drop this column, we won't need this in vm (in fact we don't use it in baremetal as well but that's a later thing)